client/src/GamemasterInGame/PlayerListComponents/PlayerActions.tsx

```diff
@@ -109,23 +122,52 @@ function RoleChangeMenuItem({
   player: string;
 }) {
   const [, , , setPlayerRole] = useAssignRole();
+  const getPlayerAlignment = useGetPlayerAlignment();
+  const alignment = getPlayerAlignment(player);
+  const role = game.playersToRoles[player];
 
   return (
     <RoleSelect
```

Comment:
Actually I want header to be simple data and move role select back down into the menu list